src/js/plugin.js

```diff
@@ -305,8 +306,10 @@ class Plugin {
       throw new Error(`Illegal plugin name, "${name}", must be a string, was ${typeof name}.`);
     }
 
-    if (pluginExists(name) || Player.prototype.hasOwnProperty(name)) {
-      throw new Error(`Illegal plugin name, "${name}", already exists.`);
+    if (pluginExists(name)) {
+      log.warn(`A plugin named "${name}" already exists. You may want to avoid re-registering plugins!`);
+    } else if (Player.prototype.hasOwnProperty(name)) {
```

Comment:
This is an `else if` because if a plugin exists, it will have a method on the `Player.prototype`, but if the plugin _does not exist_ **and** the name matches a `Player.prototype` method, it's an actual player method.